dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/models/project.py

```diff
@@ -31,11 +31,39 @@ def __init__(
         self,
         name: str = configuration.WORKFLOW_PROJECT,
         description: Optional[str] = None,
+        code: Optional[str] = None,
     ):
         super().__init__(name, description)
+        self.code = code
 
     def create_if_not_exists(self, user=configuration.USER_NAME) -> None:
         """Create Project if not exists."""
-        JavaGate().create_or_grant_project(user, self.name, self.description)
+        _ = JavaGate().create_or_grant_project(user, self.name, self.description)
         # TODO recover result checker
         # gateway_result_checker(result, None)
+
+    def get_project_by_name(self, user=configuration.USER_NAME, name=None) -> None:
+        """Get Project by name."""
+        project = JavaGate().query_project_by_name(user, name)
+        self.name = project.getName()
+        self.description = project.getDescription()
+        self.code = project.getCode()
+        return
```

Comment:
should we remove statement `return` if we return nothing?

Reply:
and think maybe we should return something here, like return the project object, 
```suggestion
    def get_project_by_name(self, user=configuration.USER_NAME, name=None) -> Project:
         """Get Project by name."""
         project = JavaGate().query_project_by_name(user, name)
         self.name = project.getName()
         self.description = project.getDescription()
         self.code = project.getCode()
         return self
```
I thing use the `get` keyword but do not return anything is odd, WDTY?